atomic_reactor/utils/operator.py

```diff
@@ -734,6 +734,28 @@ def _replace_pullspecs_not_in_annotations(self, obj, k_or_i, replacements):
             # in replacement_pullspecs and no replacement will be done
             self._replace_unnamed_pullspec(obj, k_or_i, replacements)
 
+    def _mods_append(self, append_mods):
+        if not append_mods:
+            return
+        for mod in append_mods:
+            for key in append_mods[mod]:
+                item = append_mods[mod][key]
```

Comment:
you have to first check if key exists ('skip' doesn't even have to be there)

if key exists then check if it is list => append, or None/empty  => assign list

Reply:
@rcerven; I took this: "In the example above, there could already be a “skips” array or it could be null" as referring the CSV file being modified, not the *modification* file.